doc/api/timers.md

```diff
@@ -81,6 +81,15 @@ added: v11.0.0
 
 If true, the `Timeout` object will keep the Node.js event loop active.
 
+### `timeout.close()`
+<!-- YAML
+added: v0.9.1
+-->
```

Comment:
```suggestion
-->

> Stability: 3 - Legacy: Use [`clearTimeout()`][] instead.
```

Reply:
Could you explain why do you with mark this as legacy? I don't see any deprecated warning at the code.

Reply:
I don't have any strong feeling about this. It's a method that has never been documented, it's more or less an alias to `clearTimeout`, it seems to me that if it didn't exist, we wouldn't want to create it, and we only keeping it to avoid an unnecessary breaking change. Do you plan on using it yourself?

Reply:
Yes. I use this at some projects.

Reply:
Let's wait and see if anyone else has an opinion on this.

Reply:
Marking an API as legacy does *not* mean that runtime deprecation is required. Legacy as a very specific meaning in Node.js that includes it will not be deprecated.

Reply:
btw, +1 to marking this legacy. I could also include a clear strong statement in here that `clearTimeout()` should be used instead.